src/DependencyInjection/EzPlatformHttpCacheExtension.php

```diff
@@ -69,28 +66,4 @@ public function getExtraConfigParsers()
     {
         return $this->extraConfigParsers;
     }
-
-    /**
-     * Overrides default header name based on setting tag_mode in FosHttpCacheBundle configuration.
-     */
-    private function setDefaultResponseHeader(ContainerBuilder $container): void
-    {
-        $purgeType = $container->getParameter('ezpublish.http_cache.purge_type');
-
-        $responseHeader = 'http' === $purgeType ? 'xkey' : TagHeaderFormatter::DEFAULT_HEADER_NAME;
```

Comment:
Side note: This probably need adoption in 0.9 and up as well then after @vidarl introduced `varnish` as alias to me more clear on varnish vs fastly.

Reply:
Ignore, this is master only logic.

Good that you remove it 👍